lib/travis/logs/drain_consumer.rb

```diff
@@ -161,26 +161,71 @@ def dead?
         end
       end
 
-      private def receive(delivery_info, _properties, payload)
+      private def receive(delivery_info, properties, payload)
         return if dead?
+
+        request_started_at = Time.now
+
+        delivery = delivery_info.to_hash.dup
+        delivery.delete(:delivery_tag)
+        delivery.delete(:consumer)
+        delivery.delete(:channel)
+        delivery[:delivery_tag] = delivery_info.delivery_tag
```

Comment:
This looks like it's deleted and immediately readded. Is it just a formatting issue or something else?

Reply:
Ah yeah, this one is a bit confusing. The one that is deleted is an instance of [`Bunny::VersionedDeliveryTag`](https://github.com/ruby-amqp/bunny/blob/master/lib/bunny/versioned_delivery_tag.rb), whereas the one that is re-added [is a string](https://github.com/ruby-amqp/bunny/blob/master/lib/bunny/delivery_info.rb#L72). That conversion is made in order to get a string value that can be JSON encoded and sent to honeycomb.